app/src/main/java/org/simple/clinic/bp/history/BloodPressureHistoryScreen.kt

```diff
@@ -111,14 +113,24 @@ class BloodPressureHistoryScreen(
     }
   }
 
+  private fun handleToolbarBackClick() {
+    toolbar.setNavigationOnClickListener {
+      screenRouter.pop()
+    }
+  }
+
   override fun showBloodPressureHistory(bloodPressures: List<BloodPressureMeasurement>) {
     bloodPressureHistoryAdapter.submitList(BloodPressureHistoryListItem.from(bloodPressures, config.bpEditableDuration, utcClock))
   }
 
   override fun openBloodPressureEntrySheet() {
+    val intent = BloodPressureEntrySheet.intentForNewBp(context, delegate.model.patientUuid)
```

Comment:
Can we pass the `patientUuid` as a parameter to this function instead of reading it from the `MobiusDelegate`. The `model` is currently exposed from the delegate only to support refactoring the `PatientSummaryScreen` to mobius.